scripts/ofekquerysakak.js

```diff
@@ -0,0 +1,93 @@
+(function () {
+
+//The query option itself.
+function $(query){
+    return new ofekQuery(query);
+}
+
+var ofekQuery = function(query) {
+    this.resultsFromQuery = [] ;
+    addClass(className);
+    removeClass(className);
+    var splitedElm = query.split(' ');
```

Comment:
Unnecessary cases split. Try thinking of a way you're doing it in a whole iteration instead of doing the first 1 at first